controllers/drplacementcontrol_controller_test.go

```diff
@@ -893,6 +899,17 @@ var _ = Describe("DRPlacementControl Reconciler", func() {
 				Expect(userPlacementRule.Status.Decisions[0].ClusterName).To(Equal(EastManagedCluster))
 			})
 		})
+
+		When("Deleting user PlacementRule", func() {
+			It("Should cleanup DRPC", func() {
+				// ----------------------------- DELETE DRPC from PRIMARY --------------------------------------
+				By("\n\n*** DELETE User PlacementRule ***\n\n")
+				deleteUserPlacementRule()
```

Comment:
For some reason, deleting the User PlacementRule from this unit test does not trigger a notification.  Still investigating.